test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -752,7 +762,11 @@ describe("UserWallet - Lightning Pay", () => {
 
     // await sleep(1000)
 
-    await userWallet1.getBalances()
+    const _ = await Wallets.getBalanceForWallet({
```

Comment:
if you remove this the tests fail? I think is just to update payments/invoices

Reply:
All pass still 👍🏾